app/views/splash/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<header>
+<h1>MoneyWallet</h1>
+<%= link_to 'Sign up', new_user_registration_path, class: 'sign-in-btn'%>
+</header>
+
+<section class="splash-page">
+<h1> Welcome To Money Wallet </h1>
+<p>The Best Way To Manage Your Money</p>
+<div class= "splash-actions">
+<%= link_to 'Log-in', user_session_path, class: 'sign-in-btn' %>
+</div>
+</section>
```

Comment:
- [ ] Please kindly ensure that your design matches the original design guidelines. At the moment, the buttons in the splash screen, login and signup pages do not match the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/189246470-045482ba-1cf1-4378-93fb-096d59fe98f8.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/189246566-927dc28e-c3b0-48f2-92e5-ee5101e0841b.png)